Sources/KIF/Classes/KIFUIViewTestActor.m

```diff
@@ -411,7 +411,20 @@ - (void)activateCustomActionWithName:(NSString *)name expectedResult:(BOOL)expec
             return KIFTestStepResultFailure;
         }];
     }
+}
 
+- (void)performAccessibilityActivateWithExpectedResult:(BOOL)expectedResult;
+{
+    @autoreleasepool {
+        KIFUIObject *found = [self _predicateSearchWithRequiresMatch:YES mustBeTappable:YES];
```

Comment:
Does `mustBeTappable` need to be true on these objects? I suppose you still want to limit this to objects that would presumably pass a hitTest check?

Reply:
In the cases I'm imagining they'd be tappable yes. 
I'm struggling to imagine a non tappable element that would be activated, interactive elements tend to be.

Reply:
Came back around on this and i dont think this needs to be tappable.